src/web_app_skeleton/.github/workflows/ci.yml.ecr

```diff
@@ -0,0 +1,114 @@
+name: <%= project_name %> CI
+
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: "*"
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    container:
+      image: crystallang/crystal:<%= Crystal::VERSION %>
+    steps:
+      - uses: actions/checkout@v2
+      - name: Format
+        run: crystal tool format --check
+
+  specs:
+    runs-on: ubuntu-latest
+    container:
+      image: crystallang/crystal:<%= Crystal::VERSION %>
+    env:
+      LUCKY_ENV: test
+      DB_HOST: postgres
+
+    services:
+      postgres:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+       # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install PostgreSQL client
+      run: |
+        apt-get update
+        apt-get -yqq install libpq-dev postgresql-client
+<%- if browser? -%>
+    - name: Install Chromedriver
```

Comment:
I don't think this step is required since the shard we use for webdrivers will install and add the appropriate binary to your home directory.

Reply:
Nevermind, it didn't pass without this :(